apps/web/src/components/invites/MembersTable.tsx

```diff
@@ -55,7 +56,7 @@ export function MembersTable({
             <Avatar style={{ marginRight: 10, width: 40, height: 40 }} src={member.user?.profilePicture} radius="xl">
               {capitalize((member.user?.firstName || '')[0])} {capitalize((member.user?.lastName || '')[0])}
             </Avatar>
-            <Group direction="column" spacing={5}>
+            <Group style={{ flexDirection: 'column' }} spacing={5}>
```

Comment:
Would be better to use `Stack` here
```suggestion
            <Stack spacing={5}>
```